sklearn/externals/joblib/externals/loky/cloudpickle_wrapper.py

```diff
@@ -2,7 +2,7 @@
 from functools import partial
 
 try:
-    from sklearn.externals.joblib.externals.cloudpickle import dumps, loads
+    from ..cloudpickle import dumps, loads
```

Comment:
unsure about these ones here @ogrisel @tomMoral

Reply:
anything joblib related can be ignored since we're unvendoring #13531